src/test/java/uk/gov/justice/hmpps/prison/repository/BookingRepositoryTest.java

```diff
@@ -230,7 +230,7 @@ public void testGetLatestBookingByOffenderNoHavingInactiveBooking() {
 
         assertThat(summary.getOffenderNo()).isEqualTo(offenderNoWithInactiveBooking);
         assertThat(summary.getFirstName()).isEqualTo("RICHARD");
-        assertThat(summary.getMiddleNames()).isNull();
+        assertThat(summary.getMiddleNames()).isNullOrEmpty();
```

Comment:
This is a consequence of the HSQLDB change. On NOMIS this does return null as expected, but on HSQLDB it returns empty.